AirCasting/Utils/Strings.swift

```diff
@@ -890,6 +894,33 @@ struct Strings {
                                                                      comment: "")
         static let failedSavingButton: String = NSLocalizedString("Got it!",
                                                                     comment: "")
+        static let firstDeletingAccountTitle: String = NSLocalizedString("Delete account?",
+                                                                    comment: "")
+        static let firstDeletingAccountMessage: String = NSLocalizedString("Would you like to delete your account?",
+                                                                      comment: "")
+        static let firstConfirmingDeletingButton: String = NSLocalizedString("Confirm",
+                                                                          comment: "")
+        static let secondDeletingAccountTitle: String = NSLocalizedString("Delete confirmation",
+                                                                          comment: "")
+        static let secondDeletingAccountMessage: String = NSLocalizedString("You will lose all your sessions and data. Are you sure to delete the account?",
+                                                                            comment: "")
+        static let secondConfirmingDeletingButton: String = NSLocalizedString("Delete",
+                                                                              comment: "")
+        
+        static let failedDeletingMessage: String = NSLocalizedString("Something went wrong deleting the account. Please try again later.",
+                                                                     comment: "")
+        static let failedButton: String = NSLocalizedString("Got it!",
+                                                                   comment: "")
+        static let unableConnectTitle: String = NSLocalizedString("No Internet connection",
+                                                                  comment: "")
+        static let unableConnectMessage: String = NSLocalizedString("To sign out, you must be connected to the Internet",
+                                                                    comment: "")
+        static let thirdDeletingAccountTitle: String = NSLocalizedString("Success",
+                                                                   comment: "")
+        static let thirdDeletingAccountMessage: String = NSLocalizedString("Your account has been deleted.",
+                                                                   comment: "")
+        static let thirdConfirmingDeletingButton: String = NSLocalizedString("Got it!",
+                                                                   comment: "")
```

Comment:
This "Got it" appears so many times that I think we could move it to the `Common` at the top of that file. Just an idea, apply if you want 😄

Reply:
This is a great idea. Removed unnecessary code.